modules/backup-and-restore/pages/configurations.adoc

```diff
@@ -58,8 +62,6 @@ It's recommended to keep the default value `10`, which means the number of proce
 | "DefaultCompression"
 |===
 
-IMPORTANT: If `System.Backup.Local.Enable` is set to `true`, this also enables a daily full backup at 12:00am UTC.
```

Comment:
the daily backup is only supported by  our own cloud platform. it is not available for normal customers. remove this line to avoid misleading